src/rustdoc-json-types/lib.rs

```diff
@@ -52,11 +52,36 @@ pub struct Crate {
     pub paths: HashMap<Id, ItemSummary>,
     /// Maps `crate_id` of items to a crate name and html_root_url if it exists.
     pub external_crates: HashMap<u32, ExternalCrate>,
+    /// Information about the target for which this documentation was generated
+    pub target: Target,
     /// A single version number to be used in the future when making backwards incompatible changes
     /// to the JSON output.
     pub format_version: u32,
 }
 
+/// Information about a target
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct Target {
+    /// A list of `#[target_feature]` which exist for this target, along with their status in this
+    /// compiler session
+    pub target_features: Vec<TargetFeature>,
+}
+
+/// Information about a `#[target_feature]`
```

Comment:
This could be alot more useful. This comment makes it sound like it's info about a single `#[target_feature]` annotation, but it's actually about a feature that can be used in one of those annotations (and elsewhere).

It's worth elaborating more here, and linking to other docs.

Reply:
Agreed.